src/directive/sort.directive.ts

```diff
@@ -27,23 +28,15 @@ export class JhiSortDirective {
     @Input() ascending: boolean;
     @Input() callback: Function;
 
-    sortIcon = 'fa-sort';
-    sortAscIcon = 'fa-sort-up';
-    sortDescIcon = 'fa-sort-down';
-    sortIconSelector = 'span.fa';
-
     @Output() predicateChange: EventEmitter<any> = new EventEmitter();
     @Output() ascendingChange: EventEmitter<any> = new EventEmitter();
 
     element: any;
+    config: JhiModuleConfig;
 
     constructor(el: ElementRef, renderer: Renderer, configService: JhiConfigService) {
```

Comment:
I think `renderer` is not needed here too

Reply:
it's done